index.html

```diff
@@ -133,7 +133,7 @@ <h2>Important Dates</h2>
                             <table style="width:50%">
                                 <tr>
                                     <td>Paper submission</td>
-                                    <td>July 16, 2021 (No deadline extension or grace periods will be provided. Please plan accordingly.)</td>
+                                    <td>July 16, 2021 (No deadline extension will be provided. Please plan accordingly.)</td>
```

Comment:
Oh, I'm sorry. I meant the whole comment about no extension of the deadline. I don't think that stating it brings any value...

Reply:
I think it does. Saying that in advance will prevent people from leaving things for the last minute with the belief that there will be a little bit of more time. What inspired me for thinking about not giving deadline extensions was this: https://twitter.com/ShriramKMurthi/status/1384194506010992645

Now, the question is: do we want to give deadline extension or not? If we do, then I will remove the whole comment, but if we don't do, I think we should keep it.

Reply:
Hm, I think it would be safe for us to not mention it anyway, if authors ask for an extension, I think we should grace them (as I think we can spare one or two days for it). What do you think?

Reply:
I think that giving deadline extension or grace periods is a bad practice that leads to bad habits. However, I need to send email to the steering today for the CfP, thus I don't think we have time to discuss about this in live. Therefore, I will just accept giving grace period.